### PR TITLE
add avatars

### PR DESCRIPTION
**Pull request by shivamSspirit**

### CODE REVIEW

File: Components/avatars/avatar.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="./avatar.css" />
```
Comment (zabihshaik) [inline Components/avatars/avatar.html:8]: Better to add meta tag if possible

**Review by zabihshaik — APPROVED** [review]
Looks good.

File: Components/avatars/avatar.css [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+	position: relative;
+	font-size: larger;
+	font-weight: 600;
+}
+
+.txt-sm {
+	width: 4rem;
+	height: 4rem;
+	position: relative;
+	font-size: large;
```
Comment (ivinayakg) [inline Components/avatars/avatar.css:148]: add them in text utilities

File: Components/avatars/avatar.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+</div>
+</div>
+</code>
+</pre>
+</div>
+</div>
+</div> 
+<script type="text/javascript">
+	
+(function () {
```
Comment (ivinayakg) [inline Components/avatars/avatar.html:105]: seperate JS

File: Components/avatars/avatar.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+<div class="txt-avatar txt-lg">
+<span class="txt">sss</span>
+</div>
+<div class="txt-avatar txt-md">
+<span class="txt">sss</span>
+</div>
+<div class="txt-avatar txt-sm">
+<span class="txt">ss</span>
+</div>
+</div>
```
Comment (ivinayakg) [inline Components/avatars/avatar.html:97]: use format

**Review by ivinayakg — APPROVED** [review]

**Pull request merged**